components/Map/Map.js

```diff
@@ -0,0 +1,72 @@
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import L from "leaflet";
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function Map({ locations }) {
+  let DefaultIcon = L.icon({
+    iconUrl: "/marker.svg",
+    iconSize: [30, 30],
+    popupAnchor: [0, -15],
+  });
+  return (
+    <>
+      <MapContainer
+        center={[51.5, 10.2]}
+        zoom={6}
+        scrollWheelZoom={true}
+        style={{ height: "100%", width: "100%" }}
+      >
+        <TileLayer url="https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png" />
+
+        {locations.map((location) => (
```

Comment:
Matter of taste: You could destructure location here:

```javascript
{locations.map({id, coordinates, title, info}) => (
...
)}
```

Reply:
thanks for the hint...I will plan that for the "final" refactor